word_way/word_data.py

```diff
@@ -61,6 +61,10 @@ def save_word(target_word: str,
     for item in tree.findall('item'):
         for sense in item.findall('sense'):
             target_code = sense.findtext('target_code')
+            word = session.query(Word).filter(
+                Word.target_code == target_code).first()
```

Comment:
```suggestion
            word = session.query(Word).filter(
                Word.target_code == target_code
            ).first()
```
스타일차이긴 한데 이렇게 하면 훨씬 더 가독성이 좋아집니다..

Reply:
단어정보 쓸 것 없다면 `exists()`로 확인해도 될 것 같네요.